### PR TITLE
Update design from master-branding branch

### PR DESCRIPTION
**Pull request by rsalevsky**
crowbar

### CODE REVIEW

File: crowbar_framework/app/assets/stylesheets/content/content.scss [hunk 1]
```diff
@@ -1,3 +1,12 @@
+html {
```
Comment (tboerger) [inline crowbar_framework/app/assets/stylesheets/content/content.scss]: This block and the `body` block should be part of `global.scss`

File: crowbar_framework/app/assets/stylesheets/content/content.scss [hunk 2]
```diff
@@ -45,21 +54,21 @@
   }
 }
 
-.dropdown-menu {  
-  border-radius: 6px;
-  border: 1px solid rgba(0, 0, 0, 0.2);
-  right: -1px;
+h1, h2, h3.modal-title {
```
Comment (tboerger) [inline crowbar_framework/app/assets/stylesheets/content/content.scss]: Please write it like

```css
h1,
h2,
h3.modal-title {
  ...
}
```

File: crowbar_framework/app/assets/stylesheets/content/content.scss [hunk 3]
```diff
@@ -45,21 +54,21 @@
...
+h2 {
+  font-size: 23px;
+}
 
-  li a:hover {
-    border-radius: 3px;
-    box-shadow: 1px 1px 0 rgba(0, 0, 0, 0.4) inset,1px 1px 0 rgba(255, 255, 255, 0.1);
-    background-color: #2e3235;
-  }
+h3 {
```
Comment (tboerger) [inline crowbar_framework/app/assets/stylesheets/content/content.scss]: What about `h3.modal-title`? Should it get the same size?

File: crowbar_framework/app/assets/stylesheets/content/footer.scss [hunk 4]
```diff
@@ -33,3 +35,12 @@ footer {
...
   }
 }
+
+.bottom-line {
+  height: 7px;
+  margin-top: 10px;
+  width: 100%;
+  background: #2d2c50;
+  bottom: 0;
+  @include gradient-horizontal (#3f2c50, #502c4f, 0%, 90%)
```
Comment (tboerger) [inline crowbar_framework/app/assets/stylesheets/content/footer.scss]: I think this should be `@include gradient-horizontal(#3f2c50, #502c4f, 0%, 90%);`

File: crowbar_framework/app/assets/stylesheets/content/header.scss [hunk 5]
```diff
@@ -1,35 +1,21 @@
 .navbar-inverse {
   .navbar-brand {
-    padding: 4px 15px 0;
```
Comment (tboerger) [inline crowbar_framework/app/assets/stylesheets/content/header.scss]: No padding on the side anymore? Really?

File: crowbar_framework/app/assets/stylesheets/content/header.scss [hunk 6]
```diff
@@ -1,35 +1,21 @@
...
-  .dropdown-menu {
-    border-radius: 6px;
-    border: 1px solid rgba(0, 0, 0, 0.2);
-    right: -1px;
-
-    & > li > a {
-      padding: 3px 10px;
-      margin: 0px 5px;
-    }
+  #nodegroups .panel .panel-heading {
```
Comment (tboerger) [inline crowbar_framework/app/assets/stylesheets/content/header.scss]: Why only for `#nodegroups`?

File: crowbar_framework/app/views/layouts/application.html.haml [hunk 7]
```diff
@@ -21,13 +21,11 @@
       = javascript_include_tag "ie"
 
   %body{ :class => params[:controller] }
-    %header
```
Comment (tboerger) [inline crowbar_framework/app/views/layouts/application.html.haml]: Why did you remove that? And the section as well...

**Comment by tboerger** [comment]
:+1:

**Comment by rsalevsky** [comment]
The original change is from Ken, I just reviewed the changes and opened the PR. So let's merge...
